src/watch.js

```diff
@@ -0,0 +1,112 @@
+import { configRead, configAddChangeListener } from './config';
+import './watch.css';
+
+class Watch {
+  #watch;
+  #timer;
+  #domObserver;
+  #attrChanges;
+  #PLAYER_SELECTOR = 'ytlr-watch-default';
+
+  constructor() {
+    this.createElement();
+    this.startClock();
+  }
+
+  createElement() {
+    this.#watch = document.createElement('div');
+    this.#watch.className = 'webOs-watch';
+    document.body.appendChild(this.#watch);
+  }
+
+  startClock() {
+    const nextSeg = (60 - new Date().getSeconds()) * 1000;
+
+    const setTime = () => {
+      this.#watch.innerText = new Intl.DateTimeFormat(navigator.language, {
```

Comment:
Creating an `Intl.DateTimeFormat` is an expensive operation. You should be storing this object. See this for details: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString